spec/rest/push.test.js

```diff
@@ -10,6 +10,7 @@ define(['ably', 'shared_helper', 'async'], function(Ably, helper, async) {
 		defaultHeaders = Utils.defaultPostHeaders('msgpack'),
 		testDevice = {
 			id: 'testId',
+			clientId: 'testClientId',
```

Comment:
Is this related to this PR?

Reply:
not really, was going to push it straight to master